modules/nf-core/last/dotplot/main.nf

```diff
@@ -22,9 +23,12 @@ process LAST_DOTPLOT {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def annot_a_arg = annot_a ? "-a ${annot_a}" : ''
+    def annot_b_arg = annot_b ? "-b ${annot_b}" : ''
     """
     last-dotplot \\
         $args \\
+        $annot_a_arg $annot_b_arg \\
```

Comment:
Can you put them onto separate lines please.

Reply:
Done in 0166d15b22e93d568c86623c21288326e0a629d4